api/src/main/java/marquez/MarquezApp.java

```diff
@@ -107,6 +113,12 @@ public void registerResources(
     final MarquezContext context =
         MarquezContext.builder().jdbi(jdbi).tags(config.getTags()).build();
 
+    if (config.getGraphql().isEnabled()) {
+      env.servlets()
+          .addServlet("api/v1/graphql", context.getGraphqlServlet())
```

Comment:
Mind using `/graphql` as the root? Looks like [_GraphQL takes a strong opinion on avoiding versioning_](https://graphql.org/learn/best-practices/#versioning).

Reply:
Look over the docs it wasn't made obviously clear on the best approach to version graphql endpoints (if at all). Thanks for insight 👍